frontend/src/Judge/components/ShowCard.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import Moment from 'react-moment'
+import { Button, Row, Col } from 'reactstrap'
+
+const Card = styled.div`
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  margin-top: 15px;
+  padding: 10px;
+  width: 100%;
+`
+const ButtonContainer = styled.div`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  width: '150px'
+`
+
+const ShowCard = props => (
+  <Card>
+    <h2>
+      <Link to={`show/${props.id}`}>{props.name}</Link>
```

Comment:
This was changed as per request of Bill's to be a query param, I believe the link to would look like this:  <Link to={`/submit?to=${forShow.id}`}>

Reply:
But this is for judges. They won't be submitting to a show. I'll just remove the link I guess.

Reply:
Never mind. Bill's comment `/show/<show_id>/` should still work here